docs/en/operations/server-configuration-parameters/settings.md

```diff
@@ -2019,12 +2307,104 @@ Keys:
 
 **Recommended usage**
 
-``` xml
+```xml
 <send_crash_reports>
     <enabled>true</enabled>
 </send_crash_reports>
 ```
 
+## ssh_server
```

Comment:
If we document `<ssh_server>`, we also need to document server setting `<tcp_ssh_port>`.